unitTests/ui/SliderTest.ts

```diff
@@ -77,6 +77,37 @@ export function SliderTest() {
         expect(slider.getValues()).toEqual([25, 75]);
       });
 
+      it('should return round values when using dateField', () => {
```

Comment:
Tests here are a bit lazy and do not really respect encapsulation, but I think it's fine because this allow the test to get to the point and test actually what's wrong without having over the top setup.